src/sagemaker_sklearn_extension/externals/automl_transformer.py

```diff
@@ -89,6 +89,12 @@ def transform(self, X):
             y = X[:, self.header.target_column_index]
             y_transformed = self.label_transform(y)
             non_nan_indices = np.arange(y_transformed.shape[0])[~np.isnan(y_transformed)]
+            if non_nan_indices.size == 0:
+                raise ValueError(
+                    "All response values in batch are non-numeric. With a Regression problem type only"
+                    " numeric values are used. Verify the target column is set correctly or adjust the"
+                    " problem type and retry."
```

Comment:
Nit: I'd rephrase this error message to clarify what you're referring to exactly. Maybe clarify the second sentence that the target column should only contain numeric values? Does this mean that we'll need to retrain the model (i.e. start a whole new Autopilot job)?